packages/display/src/Container.ts

```diff
@@ -465,12 +464,12 @@ export class Container extends DisplayObject
             // TODO: filter+mask, need to mask both somehow
             if (child._mask)
             {
-                const maskObject = (child._mask as MaskData).maskObject || (child._mask as Container);
+                const maskObject = ((child._mask as MaskData).maskObject || child._mask) as Container;
 
                 // TODO: maskObject is getting PIXI.DisplayObject and DisplayObject confused.
                 // Once types are sorted we wont need to do this weird conversion
```

Comment:
We should be able to remove this comment now. I added it when types where being weird for me, this fixes that